content/docs/create-a-new-react-app.md

```diff
@@ -1,94 +1,94 @@
 ---
 id: create-a-new-react-app
-title: Create a New React App
+title: Stwórz nową aplikację w Reakcie
 permalink: docs/create-a-new-react-app.html
 redirect_from:
   - "docs/add-react-to-a-new-app.html"
 prev: add-react-to-a-website.html
 next: cdn-links.html
 ---
 
-Use an integrated toolchain for the best user and developer experience.
+Najwygodniej będzie ci się pracowało jeśli zastosujesz zintegrowany zestaw narzędzi (ang. toolchain). Użycie go pozwoli ci też zapewnić dobre doświadczenie użytkownikom twojej aplikacji.
 
-This page describes a few popular React toolchains which help with tasks like:
+Poniżej przedstawiamy kilka popularnych zestawów narzędziowych używanych w Reakcie. Pomagają one w zadaniach typu:
 
-* Scaling to many files and components.
-* Using third-party libraries from npm.
-* Detecting common mistakes early.
-* Live-editing CSS and JS in development.
-* Optimizing the output for production.
+* Zwiększanie skali projektu do wielu plików i komponentów.
+* Stosowanie dodatkowych bibliotek dostępnych w npm.
+* Wczesne wykrywanie często popełnianych błędów.
+* Ciągła, automatyczna edycja CSS i JS w trakcie tworzenia aplikacji.
```

Comment:
Ta edycja raczej nie jest "automatyczna". Chodzi o to, że zmiany w plikach są na żywo odzwierciedlane na stronie, bez konieczności przeładowywania jej.